### PR TITLE
Login screen layout

### PR DESCRIPTION
**Pull request by EmmanuelWLX**
## Summary
- Layout of login screen
- Added reactotron config
- CustomTextInput component

Note: The screen was tested as a third tab in the tab navigator

### Android
login-nexus6p-1
login-nexus6p-2

### iOS
login-iPhone6-1
login-iPhone6-2

## Trello card
https://trello.com/c/6BdRthWF/24-n-921-login-maquetado

### CODE REVIEW

File: LibraryApp/App.js [hunk 1]
```diff
@@ -2,8 +2,11 @@ import React from 'react';
 
 import App from './src/app';
 
+if (__DEV__) {
+  // eslint-disable-next-line no-console
+  import('./ReactotronConfig').then(() => console.log('Reactotron Configured'));
```
Comment (mmamaniwolox) [inline LibraryApp/App.js]: Do you need the `then....` part?

File: LibraryApp/src/app/AppContainer.js [hunk 2]
```diff
@@ -38,7 +39,8 @@ const BooksNavigator = createStackNavigator(
 const TabNavigator = createBottomTabNavigator(
   {
     [Routes.Library]: BooksNavigator,
-    [Routes.EmptyScreen]: EmptyScreen
+    [Routes.EmptyScreen]: EmptyScreen,
+    [Routes.Login]: { screen: Login }
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/AppContainer.js]: Why aren't EmptyScreen and Login used the same way?

File: LibraryApp/src/app/screens/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+import Login from './layout';
+
+class LoginContainer extends Component {
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/screens/Login/index.js]: I think you dont need a container with no logic

File: LibraryApp/src/app/AppContainer.js (hunk 2 above)
Comment (EmmanuelWLX) [reply LibraryApp/src/app/AppContainer.js]: good observation, i don't know either

File: LibraryApp/App.js (hunk 1 above)
Comment (EmmanuelWLX) [reply LibraryApp/App.js]: no console? 😢 ok, it's a good point

**Pull request merged**